### PR TITLE
Add OpenTracing crate to Observability section

### PR DESCRIPTION
**Pull request by jtescher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,6 +257,7 @@ See also [Games Made With Piston](https://github.com/PistonDevelopers/piston/wik
 
 * [timberio/vector](https://github.com/timberio/vector) — A High-Performance, Logs, Metrics, & Events Router.
 * [Mnwa/gtsa](https://github.com/Mnwa/gtsa) — A simple solution to proxy gelf messages (messages for Graylog) to Sentry
+* [OpenTelemetry](https://crates.io/crates/opentelemetry) — OpenTelemetry provides a single set of APIs, libraries, agents, and collector services to capture distributed traces and metrics from your application. You can analyze them using Prometheus, Jaeger, and other observability tools. [![GitHub Actions CI](https://github.com/open-telemetry/opentelemetry-rust/workflows/CI/badge.svg)](https://github.com/open-telemetry/opentelemetry-rust/actions?query=workflow%3ACI+branch%3Amaster)
```
Comment (palfrey) [inline README.md:260]: The build failure is due to this build badge not specifying a branch. If you can fix this we can merge it.

Comment (jtescher) [reply README.md]: Hm both contain a master reference now, but still not passing?

Comment (palfrey) [reply README.md]: Build issue was a problem in master (0dcfd17f3566de395f5233bddc461ad13d8dfc73) which is now fixed

Comment (palfrey) [reply README.md]: Build issue was a problem in master (0dcfd17f3566de395f5233bddc461ad13d8dfc73) which is now fixed

**Pull request merged**